### PR TITLE
ETFE-2050 Adding CAM-NEE11

### PR DESCRIPTION
**Pull request by grahampaulcook**

### CODE REVIEW

File: conf/messages.cy [hunk 1]
```diff
@@ -213,13 +225,19 @@ consigneeExport.checkYourAnswers.label = Goods being exported outside of the UK
...
+consigneeExportVat.heading = Do you know the VAT or EORI number of the person representing the consignor at the office of export?
+consigneeExportVat.yesVatNumber = Yes - VAT number
+consigneeExportVat.yesEoriNumber = Yes - EORI number
+consigneeExportVat.no = No
+consigneeExportVat.checkYourAnswers.label = VAT or EORI number of the person representing the consignor
+consigneeExportVat.error.required = Select consigneeExportVat
+consigneeExportVat.change.hidden = VAT or EORI number of the person representing the consignor
+consigneeExportVat.vatNumber.label = VAT number
+consigneeExportVat.eoriNumber.label = EORI number
+consigneeExportVat.consigneeExportType.error.required = Select 'no' if you don't know the VAT or EORI number
```
Comment (stephenkward) [inline conf/messages.cy:238]: the apostrophes are not showing on the page when compiled.

File: conf/messages.en [hunk 2]
```diff
@@ -273,13 +285,20 @@ consigneeExport.checkYourAnswers.label = Goods being exported outside of the UK
...
+consigneeExportVat.heading = Do you know the VAT or EORI number of the person representing the consignor at the office of export?
+consigneeExportVat.yesVatNumber = Yes - VAT number
+consigneeExportVat.yesEoriNumber = Yes - EORI number
+consigneeExportVat.no = No
+consigneeExportVat.checkYourAnswers.label = VAT or EORI number of the person representing the consignor
+consigneeExportVat.error.required = Select consigneeExportVat
+consigneeExportVat.change.hidden = VAT or EORI number of the person representing the consignor
+consigneeExportVat.vatNumber.label = VAT number
+consigneeExportVat.eoriNumber.label = EORI number
+consigneeExportVat.consigneeExportType.error.required = Select 'no' if you don't know the VAT or EORI number
```
Comment (stephenkward) [inline conf/messages.en:298]: the apostrophes are not showing on the page when compiled.

**Review by stephenkward — CHANGES_REQUESTED** [review]

File: conf/messages.cy (hunk 1 above)
Comment (grahampaulcook) [reply conf/messages.cy]: https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/38/commits/ee10b3daa3c5a2b53c40c937f6ebd6493c4d3b2f fixes this

File: conf/messages.en (hunk 2 above)
Comment (grahampaulcook) [reply conf/messages.en]: https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/38/commits/ee10b3daa3c5a2b53c40c937f6ebd6493c4d3b2f fixes this

**Comment by grahampaulcook** [comment]
test this please

**Review by stephenkward — APPROVED** [review]

**Pull request merged**